authors/apps/comments/models.py

```diff
@@ -0,0 +1,39 @@
+from django.db import models
+from mptt.models import MPTTModel, TreeForeignKey
+from authors.apps.profiles.models import Profile
+
+from authors.apps.articles.models import Article
+
+
+class Comment(MPTTModel):
+
+    body = models.TextField()
+
+    article = models.ForeignKey(
+        Article,
+        related_name='comments',
+        on_delete=models.CASCADE)
+
+    author = models.ForeignKey(
+        Profile,
+        related_name='comments',
+        on_delete=models.CASCADE)
+
+    parent = TreeForeignKey(
```

Comment:
For rare applications of code, I would ask you to add some description in a comment for the rest of the team members to follow behind closely and also not struggle in case they work on a feature that directly relates with this model

Reply:
Thanks for the suggestion, this is noted.

Reply:
This has been addressed. Kindly let me know whether it is sufficient.